src/applications/personalization/profile360/containers/PaymentInformation.jsx

```diff
@@ -131,11 +174,18 @@ class PaymentInformation extends React.Component {
     } else {
       const paymentAccount = paymentInformation.responses[0].paymentAccount;
 
+      // Cannot use arrow-function below in jsx attribute to pass
```

Comment:
You can get rid of these comments now :-p

Reply:
Removed.  :D